clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/config/GoogleConfigurationProperties.groovy

```diff
@@ -30,11 +30,13 @@ class GoogleConfigurationProperties {
     List<String> imageProjects
     ConsulConfig consul
     String userDataFile
+    List<String> regions
   }
 
   List<ManagedAccount> accounts = []
   int asyncOperationTimeoutSecondsDefault = ASYNC_OPERATION_TIMEOUT_SECONDS_DEFAULT
   int asyncOperationMaxPollingIntervalSeconds = ASYNC_OPERATION_MAX_POLLING_INTERVAL_SECONDS
   List<String> baseImageProjects
   long maxMIGPageSize = 50
+  List<String> defaultRegions
```

Comment:
Same for the `regions` attribute up above.

Also, if the `regions` and `defaultRegions` attributes have implications only for indexing, the names should probably change. Is it 'managedRegions', `regionsToIndex`, or something else?